jetty-util/src/main/java/org/eclipse/jetty/util/component/Dumpable.java

```diff
@@ -70,6 +70,15 @@ static String dump(Dumpable dumpable)
         return b.toString();
     }
 
+    /**
+     * The description of this/self found in the dump.
+     * Allows for alternative representation of Object other then .toString()
+     * where the long form output of toString() is represented in a cleaner way
+     * within the dump infrastructure.
+     *
+     * @return the representation of self
+     */
+    default String dumpSelf() { return toString(); }
```

Comment:
Why is this needed? Should be enough to properly override `toString()` in subclasses, and it's already covered by the `else` branch below.